### PR TITLE
Create DIP-18.md

### PR DESCRIPTION
**Pull request by costgallo**
Create DIP-17.md for hosting DevCon agenda & website on Swarm

### CODE REVIEW

**Comment by danceratopz** [comment]
Hey @costgallo, I saw that you used `DIP-18` within your document, but named the doc itself `DIP-17.md`. Would you mind renaming your `DIP-17.md` to `DIP-18.md`? We also used `DIP-17` in #79. 

If you'd prefer we renamed our DIP instead, just let me know!

Thanks! :pray:

**Comment by costgallo** [comment]
> Hey @costgallo, I saw that you used `DIP-18` within your document, but named the doc itself `DIP-17.md`. Would you mind renaming your `DIP-17.md` to `DIP-18.md`? We also used `DIP-17` in #79.
> 
> If you'd prefer we renamed our DIP instead, just let me know!
> 
> Thanks! 🙏

Done, thanks!

**Review by rosee8 — APPROVED** [review]
Looks great! We will discuss this with the team asap.

File: DIPs/DIP-17.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+__Please answer questions below:__
+1. What actions are required to implement the proposal at Devcon?
+
+We need to set up a Bee light or full node with all the necessary data pinned on them. Also we need to set up a gateway-proxy that is able to:
+Provide a DNSLink (optional) so that it can serve regular DNS domains (e.g. devcon.ethereum.org) but still having the data in a format that is compatible with Swarm.
+
+These features of the gateway-proxy (DNSLink) are not yet implemented, but they are on our roadmap, they are aligned with our organizational goals and are not fundamentally difficult problems so we are confident that by the devcon they can be delivered and ready for production.Please note that we already have delivered website hosting with our current implementation, so we are confident in the excellent delivery. 
+
+This configuration should be load balanced and supplied with the appropriate postage stamps and chequebook balances. We have experience with how to do this with our gateway website and the underlying infrastructure. We are also in contact with the Ethereum Foundation currently discussing how to set up sSwarm nodes for certain Remix IDE services.
```
Comment (ligi) [inline DIPs/DIP-17.md:50]:
```suggestion
This configuration should be load balanced and supplied with the appropriate postage stamps and chequebook balances. We have experience with how to do this with our gateway website and the underlying infrastructure. We are also in contact with the Ethereum Foundation currently discussing how to set up Swarm nodes for certain Remix IDE services.
```

**Review by ligi — CHANGES_REQUESTED** [review]
Typo - otherwise LGTM

File: DIPs/DIP-17.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+DIP: DIP-18
```
Comment (ligi) [inline DIPs/DIP-17.md:2]:
```suggestion
DIP: DIP-17
```
must match the filename

File: DIPs/DIP-17.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+Edina Lovas will be responsible for the onsite organizations
+Antonio Gonzalo will be responsible for the touch&feel and the external communication of the project
+Tamàs Hathàzi will research & explore potential collaboration with other DIPs
+Costanza Gallo will act as the bridge between the DevCon team and the Swarm Association
+
+3. What other projects could this proposal be integrated with? (Bonus points for collaboration across teams :))
+The project could integrate Fair data society’s products such as Fairdrive (decentralised “dropbox”), Fairdrop (decentralised “wetransfer.com”), Dracula Markdown editor (for notes; can be saved to user’s Fairdrive), Fairphoto (decentralised gallery) or essentially any other dapp (NFT Generator) as solution is interoperable, while the underlying storage layer will be Swarm
+
+## Links & Additional Information
+__
```
Comment (ligi) [inline DIPs/DIP-17.md:81]:
```suggestion
```

File: DIPs/DIP-17.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+[Swarm website](https://www.ethswarm.org/)
+
+[Swarm whitepaper](https://www.ethswarm.org/swarm-whitepaper.pdf)
+
+[Book of Swarm ](https://www.ethswarm.org/The-Book-of-Swarm.pdf)
+
+[Bee Documentation](https://docs.ethswarm.org/docs/)__
```
Comment (ligi) [inline DIPs/DIP-17.md:88]:
```suggestion
[Bee Documentation](https://docs.ethswarm.org/docs)
```

**Review by ligi — CHANGES_REQUESTED** [review]
In general I think it is great! Only 2 nits to remove cruft.
Also the 2 examples do not work for me - this is not really a problem for merging but might be a problem getting this DIP accepted.

File: DIPs/DIP-18.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+Edina Lovas will be responsible for the onsite organizations
+Antonio Gonzalo will be responsible for the touch&feel and the external communication of the project
+Tamàs Hathàzi will research & explore potential collaboration with other DIPs
+Costanza Gallo will act as the bridge between the DevCon team and the Swarm Association
+
+3. What other projects could this proposal be integrated with? (Bonus points for collaboration across teams :))
+The project could integrate Fair data society’s products such as Fairdrive (decentralised “dropbox”), Fairdrop (decentralised “wetransfer.com”), Dracula Markdown editor (for notes; can be saved to user’s Fairdrive), Fairphoto (decentralised gallery) or essentially any other dapp (NFT Generator) as solution is interoperable, while the underlying storage layer will be Swarm
+
+## Links & Additional Information
+__
```
Comment (ligi) [inline DIPs/DIP-18.md:81]:
```suggestion
__
```

File: DIPs/DIP-18.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+[Swarm website](https://www.ethswarm.org/)
+
+[Swarm whitepaper](https://www.ethswarm.org/swarm-whitepaper.pdf)
+
+[Book of Swarm ](https://www.ethswarm.org/The-Book-of-Swarm.pdf)
+
+[Bee Documentation](https://docs.ethswarm.org/docs/)__
```
Comment (ligi) [inline DIPs/DIP-18.md:88]:
```suggestion
[Bee Documentation](https://docs.ethswarm.org/docs/)
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: DIPs/DIP-18.md (hunk 5 above)
Comment (ligi) [inline DIPs/DIP-18.md:81]:
```suggestion
```

File: DIPs/DIP-18.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
+---
+DIP: DIP-18
```
Comment (ligi) [inline DIPs/DIP-18.md:2]:
```suggestion
DIP: 18
```

**Comment by costgallo** [comment]
Thanks for the merge and suggestions, I will make changes in a new PR

**Pull request merged**